botocore/loaders.py

```diff
@@ -363,6 +364,35 @@ def load_data(self, name):
         # We didn't find anything that matched on any path.
         raise DataNotFoundError(data_path=name)
 
+    @instance_cache
+    def load_examples(self, service_name, api_version=None):
```

Comment:
Not a fan of adding new methods to the loader interface.  Note how there's no methods for paginators/waiters.  I'd prefer to have a separate object that takes a loader and uses `load_service_model`.  This keeps the loader interface small.